frontend/src/types/calendar.ts

```diff
@@ -0,0 +1,14 @@
+export type DateSelectMode = 'single' | 'range';
+
+export type MonthStatus = 'prevMonth' | 'currentMonth' | 'nextMonth';
```

Comment:
type명에서 Month와 관련있다는 것이 유추 가능해서 'Month' 접미사를 생략해도 괜찮을 것 같아요!
```suggestion
export type MonthStatus = 'prev' | 'current' | 'next';
```

Reply:
좋은 피드백인 것 같아요~ month가 중복되고 있네요.